git-subhistory.sh

```diff
@@ -267,18 +273,27 @@ subhistory_assimilate () {
 		revs_to_rewrite=ASSIMILATE_HEAD
 	fi
 
-	git filter-branch \
+        commit_count=$(git rev-list --count $revs_to_rewrite)
+
+        if test $commit_count -ne 0
+        then
```

Comment:
I see.  Tab/space issue in my editor - where I work we normally use spaces instead of hard tabs, I'll reconfigure for this.  I will fix.